### DISCUSSION TITLE
Make Grafana 8 No Data type email alerts start with [No Data]

### DISCUSSION THREAD

**Discussion by GyunCho** [grafana/grafana#37762] [Alerting]
After upgrading to Grafana 8, I noticed that No Data type email alerts don't have subjects starting with [No Data] anymore, so there's no way for me to know if the alert got fired because of no data.

Grafana 8 email alerts:
![image](https://user-images.githubusercontent.com/25306520/128929394-2f967409-c8ca-4053-8b3c-7c94c4ada36d.png)

Prior to Grafana 8:
![image](https://user-images.githubusercontent.com/25306520/128929446-0b1e933a-a3e5-4f23-aaa8-7b1cbc28282d.png)

Please make grafana 8 no data type email alerts start with [No Data] for better identification.

Thanks

**Comment by valenvb** [comment]
The whole system around the No Data/Execution error alerting options could do with some attention. It would be great if we could use that information in a Notification Policy to route to different contact points or use different templates based on the alert being a "real" alert vs no data/exception etc.

**Reply by dcharleston** [reply]
I was hoping there was some some kind of "nodata" label added to alerts that are triggered by the No Data policy but I don't see anything. I 100% agree with needing the ability to route based on if the alert was triggered because of No Data or not. Our use case specifically with our other alerting system is to route "missing data" alerts to Slack and all real alerts to both Slack and PagerDuty.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
